comparative-tests/target/surefire-reports/TEST-com.github.bazel_contrib.contrib_rules_jvm.comparative.PathologicalTest.xml

```diff
@@ -0,0 +1,58 @@
+    <property name="java.vm.specification.vendor" value="Oracle Corporation"/>
+    <property name="java.specification.name" value="Java Platform API Specification"/>
+    <property name="surefire.real.class.path" value="/Volumes/Dev/src/github.com/bazel-contrib/rules_jvm/comparative-tests/target/surefire/surefirebooter-20231108124940878_3.jar"/>
+    <property name="sun.management.compiler" value="HotSpot 64-Bit Tiered Compilers"/>
+    <property name="java.runtime.version" value="17.0.8+7-LTS"/>
+    <property name="user.name" value="shs"/>
+    <property name="path.separator" value=":"/>
+    <property name="os.version" value="14.2"/>
+    <property name="java.runtime.name" value="OpenJDK Runtime Environment"/>
+    <property name="file.encoding" value="UTF-8"/>
+    <property name="java.vm.name" value="OpenJDK 64-Bit Server VM"/>
+    <property name="java.vendor.version" value="AppleJDK-17.0.8.7.2"/>
+    <property name="localRepository" value="/Users/shs/.m2/repository"/>
+    <property name="java.vendor.url.bug" value="https://rdar.apple.com/new/1003593"/>
+    <property name="java.io.tmpdir" value="/var/folders/3r/7x4j_zmj0js6w0kl19jlmclr0000gn/T/"/>
+    <property name="java.version" value="17.0.8"/>
+    <property name="user.dir" value="/Volumes/Dev/src/github.com/bazel-contrib/rules_jvm/comparative-tests"/>
+    <property name="os.arch" value="aarch64"/>
+    <property name="java.vm.specification.name" value="Java Virtual Machine Specification"/>
+    <property name="native.encoding" value="UTF-8"/>
+    <property name="java.library.path" value="/Users/shs/Library/Java/Extensions:/Library/Java/Extensions:/Network/Library/Java/Extensions:/System/Library/Java/Extensions:/usr/lib/java:."/>
+    <property name="java.vm.info" value="mixed mode, sharing"/>
+    <property name="java.vendor" value="Apple Inc."/>
+    <property name="java.vm.version" value="17.0.8+7-LTS"/>
+    <property name="sun.io.unicode.encoding" value="UnicodeBig"/>
+    <property name="java.class.version" value="61.0"/>
+  </properties>
+  <testcase name="bootstrap(String)" classname="com.github.bazel_contrib.contrib_rules_jvm.comparative.PathologicalTest" time="0.0">
+    <skipped message="Environment variable [PATH] with value [/Users/shs/.nvm/versions/node/v16.17.0/bin:/usr/local/bin:/System/Cryptexes/App/usr/bin:/usr/bin:/bin:/usr/sbin:/sbin:/opt/homebrew/bin:/var/run/com.apple.security.cryptexd/codex.system/bootstrap/usr/local/bin:/var/run/com.apple.security.cryptexd/codex.system/bootstrap/usr/bin:/var/run/com.apple.security.cryptexd/codex.system/bootstrap/usr/appleinternal/bin:/opt/appleinternal/bin:/opt/apple/bin:/Library/Apple/usr/bin:/opt/xbs/bin:/usr/appleinternal/bin:/private/var/tanuki:/usr/local/share/dotnet:~/.dotnet/tools:/Users/shs/.cargo/bin:/Users/shs/Library/Application Support/JetBrains/Toolbox/scripts:/opt/homebrew/bin:/Users/shs/bin:/Users/shs/.adt/bin:/Users/shs/.dotnet/tools] matches regular expression [.*]"/>
```

Comment:
You may want to remove a few entries from this output

Reply:
Done.